pkg/kubelet/cm/devicemanager/pod_devices.go

```diff
@@ -321,10 +327,69 @@ func (pdev *podDevices) deviceRunContainerOptions(podUID, contName string) *Devi
 			annotationsMap[k] = v
 			opts.Annotations = append(opts.Annotations, kubecontainer.Annotation{Name: k, Value: v})
 		}
+
+		// Updates for CDI devices. This also updates the annotations.
+		cdiDevices, cdiAnnotations := getCDIDeviceInfo(
+			types.UID(podUID+"-"+contName+"-"+resourceName),
```

Comment:
Is this just something unique (i.e. it doesn't matter exactly what this is as long as it's unique)?

Reply:
Yes, it just needs to be unique per container.